### ISSUE TITLE
Incompatibility with dask 2024.12.0

### ISSUE THREAD

**Issue by toicca** [scikit-hep/coffea#1230]
In Dask 2024.12.0 there were changes to `Blockwise` class (specifically [here](https://github.com/dask/dask/pull/11568)) which causes at least `NanoEventsFactory.from_root().events()` to give 

```
Traceback (most recent call last):
  File "<stdin>", line 5, in <module>
  File "/eos/home-n/ntoikka/BTVNanoCommissioning/coff_test/lib/python3.11/site-packages/coffea/nanoevents/factory.py", line 715, in events
    events = self._mapping(form_mapping=self._schema)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/eos/home-n/ntoikka/BTVNanoCommissioning/coff_test/lib/python3.11/site-packages/uproot/_dask.py", line 296, in dask
    return _get_dak_array_delay_open(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/eos/home-n/ntoikka/BTVNanoCommissioning/coff_test/lib/python3.11/site-packages/uproot/_dask.py", line 1665, in _get_dak_array_delay_open
    return dask_awkward.from_map(
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/eos/home-n/ntoikka/BTVNanoCommissioning/coff_test/lib/python3.11/site-packages/dask_awkward/lib/io/io.py", line 637, in from_map
    dsk = AwkwardInputLayer(name=name, inputs=inputs, io_func=io_func)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/eos/home-n/ntoikka/BTVNanoCommissioning/coff_test/lib/python3.11/site-packages/dask_awkward/layers/layers.py", line 163, in __init__
    super().__init__(
TypeError: Blockwise.__init__() got an unexpected keyword argument 'dsk'
```

This can be tested and reproduced with the documentations basic example
```
import awkward as ak
from coffea.nanoevents import NanoEventsFactory, NanoAODSchema

NanoAODSchema.warn_missing_crossrefs = False

fname = "https://raw.githubusercontent.com/CoffeaTeam/coffea/master/tests/samples/nano_dy.root"
events = NanoEventsFactory.from_root(
    {fname: "Events"},
    schemaclass=NanoAODSchema,
    metadata={"dataset": "DYJets"},
).events()
```

I tested with Python 3.11.9, coffea 2024.11.0, and Dask 2024.12.0

**Comment by ikrommyd** [comment]
This is a known issue. CI caught it. There was an internal breaking change on the `dask` side. It's being worked on https://github.com/dask-contrib/dask-awkward/pull/556 with input from the dask team as well.
Also reported in https://github.com/dask-contrib/dask-awkward/issues/557

**Comment by lgray** [comment]
Fixed as of dask awkward 2024.12.1
